printnames.py

```diff
@@ -0,0 +1,10 @@
+class PrintNames:
+
+    def __init__(self):
```

Comment:
Modify the constructor to pass the values as arguments.

Reply:
Made changes